Week3/homework/step4.js

```diff
@@ -1,6 +1,12 @@
 'use strict';
 
 // Add your code here
+function createBase(num) {
+  return function (num1) {
+    let i = num + num1;
+    return i;
```

Comment:
The variable `i` doesn't really add anything. I would just simply return the result directly:

```js
return num + num1;
```